### ISSUE TITLE
Getting this when looping an array

### ISSUE THREAD

**Issue by sathio** [zackehh/it.each#4]
I'm looping several urls and after it has checked the last one this error is sent

`if (!suiteStats.tests[uid]) throw Error('Unrecognised test [' + runner.title + '] for suite [' + runner.parent + ']');
it.each(pagination, 'open page and take its links', ['element'], function(element){`

**Comment by whitfin** [comment]
Can you post a code example?

**Comment by entrptaher** [comment]
Wow, I just searched and saw this issue open for last 1 year.

```js
it.each(this.myobj, 'doStuff', (element)=>{})
```

This will fail since the it.each is not arrow compatible. 
Trying to see what can be done regarding this.

**Comment by entrptaher** [comment]
I looked into this after my comment.

It did not work even when it was not arrow function. The reason is, if the data is generated later, and the `this.myobj` is not available until you are inside the block. 

It's really an async await problem.

**Comment by whitfin** [comment]
@entrptaher yes, that’s just because of how JavaScript works. It’s not an issue with this library.
